docs/guppi/index.rst

```diff
@@ -0,0 +1,139 @@
+.. _guppi:
+
+.. include:: ../tutorials/glossary_substitutions.rst
+
+*****
+GUPPI
+*****
+
+The GUPPI format is the output of the `Green Bank Ultimate Pulsar Processing
+Instrument <https://safe.nrao.edu/wiki/bin/view/CICADA/NGNPP>`_ and any clones
+operating at other telescopes, such as `PUPPI at the Arecibo Observatory
+<http://www.naic.edu/puppi-observing/>`_.  Baseband specifically supports GUPPI
+data **taken in baseband mode**, and is based off of `DSPSR's implementation
+<https://github.com/demorest/dspsr>`_.  While general format specifications can
+be found at the `SERA Project
+<http://seraproject.org/mw/index.php?title=GBT_FIle_Formats>`_ and on `Paul
+Demorest's site <https://www.cv.nrao.edu/~pdemores/GUPPI_Raw_Data_Format>`_,
+some of the header information could be invalid or not applicable,
+particularly with older files.
+
+Baseband currently only supports 8-bit |elementary samples|.
+
+.. _guppi_file_structure:
+
+File Structure
+==============
+
+Each GUPPI file contains multiple (typically 128) |frames|, with each frame
+consisting of an ASCII :term:`header` composed of 80-character entries,
+followed by a binary :term:`payload` (or "block").  The header's length is
+variable, but always ends with "END" followed by 77 spaces.
+
+How samples are stored in the payload depends on whether or not it is
+**time-ordered**.  A time-ordered payload stores each channel's :term:`stream`
```

Comment:
I think we should change the term here too.

Reply:
Thanks for catching that!  Changed.